odoo/addons/base/models/ir_mail_server.py

```diff
@@ -533,6 +578,94 @@ def send_email(self, message, mail_server_id=None, smtp_server=None, smtp_port=N
             raise MailDeliveryException(_("Mail Delivery Failed"), msg)
         return message_id
 
+    @api.model
+    def _encapsulate_email(self, old_email, new_email):
```

Comment:
Side note: could be moved to tools (def email_encapsulate), with other email tools. Could be done in the PR targeting stable also.

Reply:
good idea :)